src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -26,7 +25,7 @@
  */
 abstract class AbstractConfigCommand extends ContainerDebugCommand
 {
-    protected function listBundles(OutputInterface $output)
```

Comment:
Because we are doing weird things with the `$output` variable. Everything will be fixed once we settle on this: https://github.com/symfony/symfony/issues/16000